src/Ilios/CoreBundle/Controller/OfferingController.php

```diff
@@ -237,6 +247,27 @@ public function putAction(Request $request, $id)
                 $code = Codes::HTTP_CREATED;
             }
 
+            // capture the values of offering properties pre-update
```

Comment:
This is repeated on 440.  Maybe belongs in the offering entity as getAlertProperties that returns an array?